### DISCUSSION TITLE
Rook ceph listing Objects in a bucket

### DISCUSSION THREAD

**Discussion by elizatlawy** [aws/aws-sdk-js#4206] [Q&A]
I have managed to PUT and GET objects into a` rook-ceph `bucket.

However, I when I'm trying to use the `listObjectsV2` function to list the objects in a specific bucket It returns me an empty array for Contents and CommonPrefixes. 

I wish to check if there's been such an issue before and if so what is the resolution.

Code example: 

```
const AWS = require('aws-sdk');
const s3Options = {
  accessKeyId: 'accessKeyId',
  secretAccessKey: 'secretAccessKey'
  endpoint: new AWS.Endpoint('endPoint'),
  sslEnabled: false,
  s3BucketEndpoint: true
};
const S3 = new AWS.S3(s3Options);
response = await S3.listObjectsV2( { Bucket: bucket }).promise();
console.log(response);
```

And the response is: 

` { Contents: [], CommonPrefixes: [] }`

**Comment by ajredniwja** [comment]
@elizatlawy can you please share steps to use rook-ceph bucket used to create and send API calls to that as well. 
Seems like a third-party issue as I am able to use `listObjectsV2` normally. 
Can you also check you have enough permissions, see: https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListObjectsV2.html
